### PR TITLE
C stdlib calls and FPCast

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: Tests/LLVMTests/IRBuilderSpec.swift [hunk 1]
```diff
@@ -284,12 +287,28 @@ class IRBuilderSpec : XCTestCase {
...
+        // CAST-NEXT: %8 = fpext float %7 to double
+        _ = builder.buildFPCast(fltVal, type: FloatType.double)
+
         // CAST-NEXT: ret i32 0
         builder.buildRet(IntType.int32.constant(0))
 
         // CAST-NEXT: }
         module.dump()
+
+        print(module)
```
Comment (CodaFi) [inline Tests/LLVMTests/IRBuilderSpec.swift]: Leftover from debugging 😄

Comment (harlanhaskins) [reply Tests/LLVMTests/IRBuilderSpec.swift]: **I intend to ship as-is.**

(thanks for noticing 😅 removing it now...)

**Comment by harlanhaskins** [comment]
@CodaFi Updated the branch

**Review by CodaFi — APPROVED** [review]

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**